cmd/require/args.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright The Helm Authors.
```

Comment:
headers should be changed unless this is directly lifted from helm?

Reply:
Yes, we have an open issue about license headers #8 it is being looked into, will fix as soon this has been clarified.

Reply:
:+1:

Reply:
I would drop it, considering that it all is boilerplate code and hardly copyrightable.

Reply:
I would even drop everything in `require/args*`, I don't think we are using it for now.